src/language/rule/abstractRule.ts

```diff
@@ -29,6 +30,8 @@ export abstract class AbstractRule implements IRule {
 
         if (Array.isArray(value) && value.length > 1) {
             ruleArguments = value.slice(1);
+        } else if (value.options) {
+            ruleArguments = arrayify(value.options);
```

Comment:
Allow the tslint.json maintainer to no be forced to put a single value in an array